JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -515,9 +514,10 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
         JSQDisplayedMessageCollectionViewCell * infoCell = [collectionView dequeueReusableCellWithReuseIdentifier:cellIdentifier forIndexPath:indexPath];
         infoCell.delegate = collectionView;
         infoCell.cellLabel.text = [infoMessage text];
-        infoCell.cellLabel.textColor = [UIColor darkGrayColor];
-        infoCell.cellLabel.layer.borderColor = [[UIColor colorWithRed:239.f/255.f green:189.f/255.f blue:88.f/255.f alpha:1.0f] CGColor];
-        infoCell.headerImageView.image = [UIImage imageNamed:@"warning_white"];
+        infoCell.cellLabel.textColor = [UIColor colorWithRed:32.f/255.f green:144.f/255.f blue:234.f/255.f  alpha:1.f];
+        infoCell.cellLabel.font = [UIFont fontWithName:@"HelveticaNeue-Medium" size:12.0f];
+        infoCell.cellLabel.layer.borderWidth = 0;
```

Comment:
we don't actually want to change the info cells globally as they will be used elsewhere, so separate logic is needed on the Signal-iOS side to handle group messages and calls from the information messages (which should not be blue). that said, we merged in some basic stuff recently to blue the messages already with that logic. there will be some tweaking there needed. I'll tag you in the issue when we create it today and speak up if you have a bit of time this week. we're in the unusual time crunch situation. starting next week we'll all be working remote so it will be easier to keep in sync.